packages/components/organisms/f-header/src/components/UserNavigationPanel.vue

```diff
@@ -89,31 +90,22 @@ export default {
 @import '../assets/scss/navigation.scss';
 
 .list-link {
-    display: block;
-    padding: spacing(c) spacing(d);
-    margin: 0;
-    font-family: $font-family-base;
-    font-weight: 300;
-    color: $color-content-subdued;
-    height: auto;
-    @include font-size('body-s');
-    text-decoration: none;
-    border-bottom: 1px solid $color-border-default;
+    @include media('>mid') {
+        display: block;
+        padding: spacing(c) spacing(d);
+        margin: 0;
+        font-family: $font-family-base;
+        font-weight: 300;
```

Comment:
Same as above – 300 isn't a font-weight in the design tokens currently

Reply:
fixed 👍 have noticed that `font-weight: 300` also exists in f-checkout so can check with designs and add a small ticket to change that to `font-weight-regular`